.github/workflows/services_check.yml

```
@@ -0,0 +1,44 @@
+name: Docker Services Check
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)